core/src/main/java/com/hcl/ecomm/core/servlets/DeleteCartItemServlet.java

```diff
@@ -68,9 +67,9 @@ protected void doPut(SlingHttpServletRequest request, SlingHttpServletResponse r
 			String payload = buffer.toString();
 			String customerToken = request.getHeader("CustomerToken");
 			if (StringUtils.isNotEmpty(payload)) {
-				JSONObject jsonPayload =  new JSONObject(payload);
+				JSONObject jsonPayload = new JSONObject(payload);
 				if (isValidItem(jsonPayload)) {
-					JSONObject deleteCartItemResponse = deleteCartItemService.deleteCartItem(jsonPayload.getString("cartId"), jsonPayload.getString("itemId"), customerToken);
```

Comment:
@Subrat-Dash-416  Why this class changed?

Reply:
@rakeshkumar-verma  As we need to surpass this method to return a json object which is required for the junit test class ,in order to cover this method call we have used spy by extending this line as method.